README.md

```diff
@@ -95,4 +94,108 @@ Running the bots manually can be done with:
 ```
 cd {bot_folder}
 (DAML_LEDGER_URL=localhost:6865 poetry run python bot/{bot_name}_bot.py
+```
+
+# Running in DABL
+These instructions will show you how to build and deploy the Marketplace to the DABL cloud service.
+
+### Build the DIT file
+First, rebuild the project.
+```
+make clean && package
+```
+
+This will build the model, UI, and bots, as well as package the project in a `.dit` file which can be uploaded to DABL.
+
+### Create, upload and launch the project
+Open http://projectdabl.com. Once you are logged in, click on "New Project":
+
+![1_create_project](https://user-images.githubusercontent.com/71082197/98857327-ec817480-242c-11eb-9bfe-972dd5b7aa7b.png)
+
+Next, enter a name and a ledger name for this project:
+
+![2_name_project](https://user-images.githubusercontent.com/71082197/98857328-ec817480-242c-11eb-9d03-8e7a0f260dd1.png)
+
+Once the project is created, click on the ledger in the project explorer. This will take you to the project's console.
+
+Under the deployments tab, click on "Upload File" and upload the `da-marketplace-0.0.2.dit` file found in the `target` folder:
```

Comment:
You might have to check with Mike (or just test it out with a burner account), but I was the under the impression that DITs are only upload-able by DA employees (or those with DA email addresses)

Reply:
good point

Reply:
Changed this to upload the files manually.